dyninstAPI/src/BPatch_image.C

```diff
@@ -935,6 +935,21 @@ char *BPatch_image::getProgramFileName(char *name, unsigned int len)
    return name;
 }
 
+std::string BPatch_image::getProgramFileName() const {
+   std::vector<AddressSpace*> addrSpaces;
+   addSpace->getAS(addrSpaces);
+   AddressSpace *addrSpace = addrSpaces[0];
```

Comment:
I know the other implementation doesn't check, but are we guaranteed to have at least one address space here?